import_export/admin.py

```diff
@@ -942,6 +944,8 @@ def get_actions(self, request):
         Adds the export action to the list of available actions.
         """
         actions = super().get_actions(request)
+        if not self.has_export_permission(request):
```

Comment:
Presumably we don't need this now - it works ok without it

Reply:
It doesn't ... I still got the export dropdown now. The button is fixed though.
![image](https://github.com/user-attachments/assets/c10a563a-c9c7-4c71-8560-66ba43452ec6)

Reply:
Ah ok - to reproduce, I needed to define `IMPORT_EXPORT_EXPORT_PERMISSION_CODE = "export"` and then remove export permission for the user.  In that case it is still shown in the dropdown.  I'll update the test.